crates/router/src/connector/adyen/transformers.rs

```diff
@@ -232,11 +275,31 @@ pub struct AdyenApplePay {
     apple_pay_token: String,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
Since all the pay_later methods have the same fields, use a common struct with name `AdyenPayLaterData` and use it for all pay_later methods